inspirehep/modules/tools/templates/authorlist_form.html

```diff
@@ -0,0 +1,74 @@
+{#
+## This file is part of INSPIRE.
+## Copyright (C) 2016 CERN.
+##
+## INSPIRE is free software; you can redistribute it and/or
+## modify it under the terms of the GNU General Public License as
+## published by the Free Software Foundation; either version 2 of the
+## License, or (at your option) any later version.
+##
+## INSPIRE is distributed in the hope that it will be useful, but
+## WITHOUT ANY WARRANTY; without even the implied warranty of
+## MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+## General Public License for more details.
+##
+## You should have received a copy of the GNU General Public License
+## along with INSPIRE; if not, write to the Free Software Foundation, Inc.,
+## 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
+##}
+
+{% from "invenio_accounts/_macros.html" import render_field %}
+{% extends config.BASE_TEMPLATE %}
+
+{% block body %}
+
+<h1>Authorlist: convert author strings to MARC</h1>
+<form method=post action="/tools/authorlist" id="authorlist">
+  <dl>
+    {{ render_field(form.author_string) }}
+  </dl>
+  <p><input type=submit value=Format>
+</form>
+
+{% if results %}
+<h4>{{ results_title }}</h4>
+<pre><textarea class="form-control" form=authorlist id=resultarea>{{ results }}</textarea></pre>
```

Comment:
@jacquerie this is the only place where user provided input is used. Isn't this automatically escaped by Jinja2 and hence safe from XSS?